proxy/proxy.go

```diff
@@ -18,7 +18,7 @@ type contextKey string
 const (
 	prefixKey contextKey = "prefix"
 
-	prefixHeader string = "X-Forwarded-Prefix"
```

Comment:
Any reason to export this in here?

Reply:
What I'm trying to do is have Observatorium replace `<base href="/">` with `<base href="/api/traces/v1/{tenant}/">` for the reasons I explain at https://github.com/observatorium/api/pull/252#discussion_r831156906

In the place I am doing this inside of httputil.ReverseProxy.ModifyResponse, and at this point the HTTP request Context doesn't have the prefix that it used.  I could try to reconstruct the value applied by `MiddlewareSetPrefixHeader()`, but instead of doing that I just lookup that value Observatorium supplied for `X-Forwarded-Prefix` which happens to be exactly the value /api/traces/v1/{tenant}/ that I need to make the UI work.

I don't need to get the value from here, I could just use the literal `"X-Forwarded-Prefix"` myself.  Or we can talk about the whole approach.  What is your preference?

Reply:
I might have misunderstood. I only meant if we need have that constant accessible from other packages (i.e. the constant name is capitalized), if we don't use the constant outside of here.